README.md

```diff
@@ -553,6 +553,84 @@ polygon_deposit_res = client.deposit_from_polygon_network_with_signer(
 )
 ```
 
+#### Cross Network Deposit
+
+There are two ways to make a deposit on the Cross network:
+
+1. Using ETH Private Key and RPC URL:<br>
+In this method, you will use an ETH private key and an RPC URL to execute a Cross deposit. You'll also need to create an RPC URL using services like Infura, Alchemy, etc. Here's the code snippet for this method:
```

Comment:
"In this method, you will use an ETH private key and an RPC URL to execute a Cross deposit". to "In this method, you will use an ETH private key and an RPC URL to execute a Cross chain/network deposit".